indigo_app/static/stylesheets/_akn.scss

```diff
@@ -27,6 +28,7 @@ $hierarchy-elems: (
   'crossHeading': $gray-500,
 );
 
+.show-akn-hierarchy la-akoma-ntoso,
 .show-akn-hierarchy .akoma-ntoso {
```

Comment:
This `.akoma-ntoso` can fall away. We only show the hierarchy in the document editor view.

Reply:
done